package.json

```diff
@@ -76,6 +73,8 @@
     "webpack-stream": "^3.2.0"
   },
   "dependencies": {
+    "@types/react": ">=15 <16",
+    "moment": "2.16.x",
```

Comment:
Actually it is dependency as it is used all over the place. When you use only version compiled by webpack then you don't really need anything as output will be just single file. However if you add this to your project dependencies in node-like environment you need to be sure the library is there and it's resolvable. So the `moment` should be at least in `peerDependencies` if not `dependencies`.